jscomp/others/belt_Option.mli

```diff
@@ -27,6 +27,22 @@
     Utilities for option data type
 *)
 
+val applyU : 'a option -> ('a -> unit [@bs]) -> unit
+
```

Comment:
Could you remove the blank line, to unambiguously attach the comment to its item. Also I think it makes sense to swap the declarations of `applyU` and `apply` in the interface file, since `applyU`'s comment refers to `apply` it makes sense for it to be declared after it.

Reply:
But the uncurried version is always above the normal one, that's why I put it there.
With everything else, I agree.